tsconfig.json

```diff
@@ -1,21 +1,30 @@
 {
-	"compilerOptions": {
-		"outDir": "dist",
-		"sourceMap": true,
-		"declaration": true,
-		"moduleResolution": "Node16",
+  "compilerOptions": {
+    "outDir": "dist",
+    "sourceMap": true,
+    "declaration": true,
+    "moduleResolution": "Node16",
 
-		"module": "ES6",
-		"target": "es2020",
-		"lib": ["es2015", "es2016", "es2017", "es2018", "es2019", "es2020", "DOM"],
```

Comment:
we do need the DOM library, at the very least for the `sync` directory which uses HTML5 APIs

Reply:
Ah yeah I see it now.

Reply:
for now I've removed the library from tsconfig, and instead used `/// <reference lib="dom" />` in `sync` files. not good but better than nothing